applications/ExaquteSandboxApplication/ExaquteSandboxApplication.json

```diff
@@ -0,0 +1,11 @@
+{
+	"wheel_name": "KratosExaquteSandboxApplication",
```

Comment:
should this be in the wheels?
I would say no as it is only for the project (isn't it over by now BTW?)

Reply:
To be honest (as you can image) I generated all this automatically. I know I removed a couple manually but if there is no interest into this to be released as an app we can safely remove it. @riccardotosi ?

Reply:
The project finishes on 30/11/2021. About having ExaquteSandboxApplication wheel, it depends on deciding if we want to be able or not run ExaQUte related examples using wheels or not. Who should decide?

Reply:
ah ok if it is automatic then sure

I am commenting as there is usually an outcry if we remove stuff, even if it was temporary to begin with

I dont mind having it it, your call

Reply:
Well in this case, we can leave it there. the `json` is not in any specific format or anything, I think @HubertBalcerzak just put the fields he needed so the `setup.py` works.

My idea was to reuse those to add more info about the apps if we ever need to, like if there are research applications, core, etc...